deep_agent/aegra/startup.py

```
@@ -264,6 +265,23 @@ async def _resume_interrupted_runs() -> str:
+async def _init_pool() -> str:
+    """Initialise the shared async PostgreSQL connection pool."""
+    try:
+        from deep_agent.src.settings import settings
+
+        if not settings.database_uri:
+            return "skipped: no database_uri configured"
+
+        from deep_agent.aegra.db import init_pool
+
+        await init_pool(settings.database_uri)
+        return "ok"
```

Ruff (0.16.1)

[warning] Consider moving this statement to an `else` block

(TRY300)